### DISCUSSION TITLE
I got rejected because i dont appear to be near the campus?

### DISCUSSION THREAD

**Discussion by Biraru** [community/community#139271] [GitHub Education]
### Hi everyone,

Hello, i had summited my application for the school education benefits but i got rejected because i am not near the school campus? i live far away to my school about a 14 minutes drive.

**Comment by rakibnsajib** [comment]
You can apply when you are in campus. It's a common procedure for verifying your current student status.

**Comment by AESMatias** [comment]
Sorry, the application request verify your current location.
If the problem goes further, you can always contact the support by sumitting a new ticket: https://support.github.com/request
Good luck!
